pkgs/applications/misc/shoutrrr/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "shoutrrr";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "containrrr";
+    repo = "shoutrrr";
+    rev = "v${version}";
+    sha256 = "sha256-DGyFo2oRZ39r1awqh5AXjOL2VShABarFbOMIcEXlWq4=";
+  };
+
+  vendorHash = "sha256-+LDA3Q6OSxHwKYoO5gtNUryB9EbLe2jJtUbLXnA2Lug=";
+
+  meta = with lib; {
```

Comment:
`with lib;` is an anti-pattern, see here https://github.com/NixOS/nixpkgs/issues/208242.

Reply:
done